src/utils/mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin other-text() {
+  line-height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  color: $other-text-color;
```

Comment:
The variable `$other-text-color` is used here. Make sure it is defined in your SCSS files to avoid any compilation issues.